### DISCUSSION TITLE
Need help with verification.

### DISCUSSION THREAD

**Discussion by HCBlackFox** [community/community#68851] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/45334041/cefdc11d-1fff-439e-9b16-c69afd3a87f7)
I provide github academic email address and also an student ID that rejected, but I dont understand why? I renamed my profile how it is say, but it doesnt work, any help?

**Comment by mantasu** [comment]
Hey, it could be a translation issue, here is an [answer](https://github.com/orgs/community/discussions/68558#discussioncomment-7163615).

**Comment by Abkeyev99** [comment]
Click "Marked as answer" Thanks so good :)
